cassandra.go

```diff
@@ -154,3 +153,46 @@ func translateConsistency(consistencyName string) (gocql.Consistency, error) {
 	}
 	return gocql.One, fmt.Errorf("unknown consistency: %v", consistencyName)
 }
+
+func translateDuration(k string, df time.Duration) (time.Duration, error) {
+	if k == "" {
+		return df, nil
+	}
+	return time.ParseDuration(k)
+}
+
+func setDefaults(cfg CassandraConfig) (*gocql.ClusterConfig, error) {
+	// convert consistency name into appropriate gocql.Consistency value
+	consistency, err := translateConsistency(cfg.Consistency)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
Can you either remove this empty line or add empty lines after each `if` block below, for consistency?